.travis.yml

```diff
@@ -1,4 +1,17 @@
 language: generic
-before_install: rvm reinstall ruby-head --binary
+addons:
+  apt:
+    packages:
+      - python3
+before_install:
+  - rvm reinstall ruby-head --binary
+  - jdk_switcher use oraclejdk8
+  - ./.travis/install_from_source
```

Comment:
any objection to making the script name a bit more descriptive? I'm thinking something like: 'install_html_validator' as an alternative, notably getting the name of 'what' we are installing in the script file somewhere.